src/pages/bootcamp-dimitar/index.tsx

```diff
@@ -0,0 +1,214 @@
+import { ApiErrors } from 'common/api-errors'
+import { AlertStore } from 'stores/AlertStore'
+import { AxiosError, AxiosResponse } from 'axios'
+
+const style = {
+  position: 'absolute' as const,
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
+  boxShadow: 24,
+  p: 4,
+}
+
+function BootcampDimitarList() {
+  const { data = [] } = useBootcampDimitarList()
+  const [open, setOpen] = React.useState(false)
+  const [row, setRow] = React.useState<{ firstName: string; lastName: string; company: string }>()
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = React.useState(false)
+  const [isDeleteSelectedModalOpen, setIsDeleteSelectedModalOpen] = React.useState(false)
+  const [rowToDelete, setRowToDelete] = React.useState<{ id: string }>()
+  const [selectedRows, setSelectedRows] = React.useState<BootcampDimitarResponse[]>([])
+  const [selectionModel, setSelectionModel] = React.useState<GridRowId[]>([])
+  const router = useRouter()
+
+  const mutation = useMutation<AxiosResponse<BootcampDimitarResponse>, AxiosError<ApiErrors>, any>({
+    mutationFn: deleteBootcampDimitar,
+    onError: () => AlertStore.show('error', 'error'),
+    onSuccess: () => AlertStore.show('success', 'success'),
+  })
+
+  const handleDeleteModalOpen = (row: any) => {
+    return () => {
+      setRowToDelete(row)
+      setIsDeleteModalOpen(true)
+    }
+  }
+
+  const handleDeleteModalClose = () => {
+    setIsDeleteModalOpen(false)
+  }
+
+  const handleOpen = (row: any) => {
+    return () => {
+      setOpen(true)
+      setRow(row)
+    }
+  }
+
+  const handleClose = () => {
+    setOpen(false)
+  }
+
+  const deleteHandler = () => {
+    mutation.mutateAsync(rowToDelete?.id as string).then(() => {
+      handleDeleteModalClose()
+      router.reload()
+    })
+  }
+
+  const deleteAllHandler = () => {
+    selectedRows.forEach((row: any) => {
+      mutation.mutateAsync(row?.id as string).then((_) => {
+        router.reload()
+      })
+    })
+  }
+
+  const closeDeleteSelectedHandler = () => {
+    setIsDeleteSelectedModalOpen(false)
+    setSelectedRows([])
+    setSelectionModel([])
+  }
+
+  const columns: GridColumns = [
+    {
+      field: 'id',
+      headerName: 'ID',
+      valueGetter: (p) => p.row.id,
+      flex: 2,
+    },
+    {
+      field: 'name',
+      headerName: 'Name',
+      valueGetter: (p) => `${p.row.firstName} ${p.row.lastName}`,
+      flex: 3,
+    },
+    {
+      field: 'company',
+      headerName: 'Company',
+      valueGetter: (p) => p.row.company,
+      flex: 3,
+    },
+    {
+      field: 'actions',
+      headerName: 'Actions',
+      renderCell: (cellValues) => {
+        return (
+          <>
+            <IconButton size="small" sx={{ mr: 1 }} onClick={handleOpen(cellValues.row)}>
+              <PageviewIcon />
+            </IconButton>
+            <Link href={`/bootcamp-dimitar/${cellValues.row.id}/edit`}>
```

Comment:
These links are better off when collected in `routes.ts` instead of hardcoded on multiple places. This will make the refactoring much easier later on